src/components/main/AirQualityCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react';
+import styles from './Main.css';
+import { AirQualityType } from 'types/airQualityType';
+import { convertTime } from '../../utils/helpers';
+
+type HourlyCardProps = {
+  airQuality: AirQualityType;
+};
+
+const AirQualityCard: FC<HourlyCardProps> = ({ airQuality }) => {
+  const {
+    time,
+    pollutant,
+    AQI,
+    AQI_CO,
+    AQI_NO2,
+    AQI_O3,
+    AQI_SO2,
+    AQI_PM10,
+    AQI_PM2P5
+  } = airQuality;
+
+  const date = convertTime(time);
+
+  return (
+    <div className={styles['hourly-card']}>
+      <h4 className={styles['hourly-card__title']}>{`${date.hours}:${date.minutes}`}</h4>
```

Comment:
i've already seen you have function `getTimeLabel`, please make this function shared and reuse everywhere